### DISCUSSION TITLE
Insert links by pasting into selected text

### DISCUSSION THREAD

**Discussion by remus-selea** [TriliumNext/Trilium#4104] [Ideas]
Maybe an extension if not a feature, similar to https://github.com/denolehov/obsidian-url-into-selection

**Comment by XXXJetfireXXX** [comment]
You mean like this?
![image](https://github.com/zadam/trilium/assets/43646178/2464f2b6-c681-491d-9a61-fab516fa4356)
![image](https://github.com/zadam/trilium/assets/43646178/8788ef03-385b-4052-b560-59d3dc72858a)

**Reply by remus-selea** [reply]
I meant pasting directly over the selected text without additional steps.

**Reply by zadam** [reply]
This would be a rather CKEditor feature.

However a CTRL-K shortcut works quite well in practice for this.
